modules/FhirAPI/src/FhirAPI/Service/FhirRequestParamsHandler.php

```diff
@@ -128,6 +129,17 @@ private function processRequestParams(){
                                 $requestParams['PARAMETERS_FOR_SEARCH_RESULT'][$name][] =$tempValue;
                             }
                             break;
+                        case "filter":
+                            $cleanValue = preg_replace("/[\W]/", "", $value);
+                            $operators=  preg_replace("/[\w]/", "", $value);
+                            if(strpos($operators, '^') !== false){
```

Comment:
where do you found the ^ operator is regular like? (I've not fount it) usually ^ equal to '%string' and not  '%string%', prefer to check again...

Reply:
I can not use % I am open to any other char ....